pipe/pressuretransducer/pressuretransducer_fill_date_gaps_and_regularize.yaml

```diff
@@ -61,17 +61,19 @@ input:
         name: DATA_PATH
         repo: pressuretransducer_location_group_and_restructure
         # For full-scale daily processing, glob should be /pressuretransducer/(*/*/*). To limit to particular CFGLOCs, note the parentheses and enter something like /pressuretransducer/(*/*/*/(CFGLOC108321|...|CFGLOC110994))
-        glob: /pressuretransducer/((*/*/*)/(CFGLOC103648|CFGLOC104730|CFGLOC105231|CFGLOC106590|CFGLOC107046|CFGLOC107047|CFGLOC107048|CFGLOC107049|CFGLOC107050|CFGLOC107051|CFGLOC107052|CFGLOC107053|CFGLOC107054|CFGLOC107055|CFGLOC107056|CFGLOC107057|CFGLOC107058|CFGLOC107059|CFGLOC107060|CFGLOC107068|CFGLOC107069|CFGLOC107070|CFGLOC107071|CFGLOC107072|CFGLOC107073|CFGLOC107074|CFGLOC107075|CFGLOC107076|CFGLOC107077|CFGLOC107078|CFGLOC107079|CFGLOC107080|CFGLOC107081|CFGLOC107082|CFGLOC109219|CFGLOC109220|CFGLOC109221|CFGLOC109222|CFGLOC109223|CFGLOC109224|CFGLOC109225|CFGLOC109226|CFGLOC109227|CFGLOC109228|CFGLOC109229|CFGLOC109230|CFGLOC109231|CFGLOC109232|CFGLOC109233|CFGLOC109241|CFGLOC109242|CFGLOC109243|CFGLOC109244|CFGLOC109245|CFGLOC109246|CFGLOC109247|CFGLOC109248|CFGLOC109249|CFGLOC109250|CFGLOC109251|CFGLOC109252|CFGLOC109253|CFGLOC109254|CFGLOC109255|CFGLOC110635|CFGLOC110636|CFGLOC110637|CFGLOC110638|CFGLOC110639|CFGLOC110640|CFGLOC110641|CFGLOC110642|CFGLOC110643|CFGLOC110644|CFGLOC110645|CFGLOC110646|CFGLOC110647|CFGLOC110648|CFGLOC110649|CFGLOC110688|CFGLOC110941|CFGLOC110942|CFGLOC110943|CFGLOC110944|CFGLOC110945|CFGLOC110946|CFGLOC110947|CFGLOC110948|CFGLOC110949|CFGLOC110950|CFGLOC110951|CFGLOC110952|CFGLOC110953|CFGLOC110954|CFGLOC110955|CFGLOC111001|CFGLOC113185|CFGLOC113186|CFGLOC113187|CFGLOC113188|CFGLOC113189|CFGLOC113190|CFGLOC113191|CFGLOC113192|CFGLOC113193|CFGLOC113194|CFGLOC113195|CFGLOC113196|CFGLOC113197|CFGLOC113198|CFGLOC113199|CFGLOC113666|CFGLOC113667|CFGLOC113668|CFGLOC113669|CFGLOC113670|CFGLOC113671|CFGLOC113672|CFGLOC113673|CFGLOC113674|CFGLOC113675|CFGLOC113676|CFGLOC113677|CFGLOC113678|CFGLOC113679|CFGLOC113680|CFGLOC113749))
+        glob: /pressuretransducer/((*/*/*)
```

Comment:
Looks like you might have an extra opening parenthesis

Reply:
Good catches! I fixed the two issues, could you please approve it to unblock the merge. Thanks!